village/src/components/SmurfForm.js

```diff
@@ -13,6 +14,20 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
+    const newSmurf = {
+      name: this.state.name,
+      age: this.state.age,
+      height: this.state.height,
+    }
+    axios
+    .post("http://localhost:3333/smurfs", newSmurf)
+    .then(response => {
+      this.setState({ smurfs: response.data });
```

Comment:
you're setting state in the wrong place. You need to send down a method on props from App that takes the response.data and uses that to update state inside App